src/zenml/cli/server.py

```diff
@@ -30,6 +30,7 @@
 from zenml.logger import get_logger
 from zenml.utils import yaml_utils
 from zenml.utils.analytics_utils import AnalyticsEvent, track_event
+from zenml.constants import ENV_AUTO_OPEN_DASHBOARD
```

Comment:
If you would like to use that function below :D

```suggestion
from zenml.constants import ENV_AUTO_OPEN_DASHBOARD, handle_bool_env_var
```